src/app/components/ui/ukhsa/Map/shared/controls/HealthAlertControl.tsx

```diff
@@ -73,8 +73,11 @@ const AlertDialogContent = () => {
             <SummaryListRow>
               <SummaryListKey>{t('map.alertDialog.statusKey')}</SummaryListKey>
               <SummaryListValue>
-                <div className={clsx(`govuk-tag capitalize`, getTagVariantFromStatus(status))}>
-                  {status.toLowerCase()}
+                <div
+                  className={clsx(`govuk-tag capitalize`, getTextColourCssFromColour(status))}
```

Comment:
We avoid using html style props on components, but wagtail should be able to handle this, potentially a classname of like

```suggestion
                  className={clsx(`govuk-tag capitalize`, getTextColourCssFromColour(status), `bg-${getCssVariableFromColour(status)} )}
```

Might do the trick - have a little experiment with these?

Reply:
I have removed all instances where style was used and replaced with classes as is the standard